### PR TITLE
OEA-405: Added a new renderModulePartial() call in the patient summar…

### PR DESCRIPTION
**Pull request by marshl**
…y screen to render the trial summary above the patient contact information

### CODE REVIEW

File: protected/views/patient/view.php [hunk 1]
```diff
@@ -92,6 +92,7 @@
 				</section>
 			<?php endif ?>
 			<?php $this->renderPartial('_patient_details')?>
+      <?php $this->renderModulePartials('patient_summary_column1_before_contact')?>
```
Comment (apigram) [inline protected/views/patient/view.php]: Alignment looks a little off here.

**Review by apigram — COMMENTED** [review]
I'm wondering if there's a better way we can modify the renderModulePartials function so we can directly specify the on-screen ordering?

File: protected/views/patient/view.php (hunk 1 above)
Comment (marshl) [reply protected/views/patient/view.php]: It's because this line uses spaces, while the other lines use tabs.

**Comment by marshl** [comment]
Changing the ordering was what I initially tried, but that won't help because of the fact that the module partials and non-module partials are treated completely separately. The only way I could think of changing the order of the non-module partials would be render them as partials in the one function. is that a change we want to make?

**Comment by apigram** [comment]
I feel as though it would give us more control over the arrangement of the sections without having to create dependencies or add more segmentation.

**Pull request closed without merging**